src/main/scala/com/campudus/tableaux/controller/MediaController.scala

```diff
@@ -225,6 +227,9 @@ class MediaController(override val config: TableauxConfig,
 
   def deleteFile(uuid: UUID, langtag: String): Future[TableauxFile] = {
     for {
+    // retrieve cells with this file for cache invalidation
+      cellsForFiles <- attachmentModel.retrieveCells(uuid)
```

Comment:
Same question as above - first retrieve / delete and then do everything to invalidate cache?